commands/apps/trello/create-trello-card.template.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+# Dependency: This script requires the following Python libraries: `sys`, `requests`, `dateparser`
+# Install them with `pip install {library}`
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Create Card
+# @raycast.packageName Trello
+# @raycast.mode compact
+
+# Optional parameters:
+# @raycast.icon 🤖
+# @raycast.argument1 { "type": "text", "placeholder": "Card title" }
+# @raycast.argument2 { "type": "text", "placeholder": "Due date (e.g. today)", "optional": true }
+
+# Documentation:
+# @raycast.description Create a new Trello card
+# @raycast.author Michael Francis
+# @raycast.authorURL https://github.com/mikefrancis
+
+
+import sys
+import requests
+import dateparser
+
+# To generate an API key/token, head to https://trello.com/app-key
+TRELLO_KEY = ''
+TRELLO_TOKEN = ''
+# To find the Board ID, head to https://api.trello.com/1/members/me/boards?key={TRELLO_KEY}&token={TRELLO_TOKEN}
+# To find the List ID, head to https://api.trello.com/1/boards/{BOARD_ID}/lists?key={TRELLO_KEY}&token={TRELLO_TOKEN}
+TRELLO_LIST_ID = ''
+
+if TRELLO_KEY == '' or TRELLO_TOKEN == '' or TRELLO_LIST_ID == '':
+  print('Command not configured correctly, missing either TRELLO_KEY, TRELLO_TOKEN or TRELLO_LIST_ID')
+  exit(1)
```

Comment:
Let's separate the conditions in three to make easier to explain the user if something is missing:

```suggestion
if TRELLO_KEY == '':
  print('Command not configured correctly. Missing variable: TRELLO_KEY')
  exit(1)

if TRELLO_TOKEN == '':
  print('Command not configured correctly. Missing variable: TRELLO_TOKEN')
  exit(1)

if TRELLO_LIST_ID == '':
  print('Command not configured correctly. Missing variable: TRELLO_LIST_ID')
  exit(1)
```